activiti-cloud-runtime-bundle-service/activiti-cloud-services-runtime-bundle/activiti-cloud-services-core/src/main/java/org/activiti/cloud/services/core/VariableValueConverter.java

```diff
@@ -26,9 +26,9 @@
 import org.springframework.core.convert.ConversionService;
 import org.springframework.util.Assert;
 
-public class ProcessVariableValueConverter {
```

Comment:
Any reason to change public class name? It will cause backward compatibility and compile issues downstream..

Reply:
Just because it now converted not just process variables, but if it can cause problems I will just restore the name.